graph/memstore/quadstore.go

```diff
@@ -143,6 +145,11 @@ type QuadStore struct {
 	index   QuadDirectionIndex
 	horizon int64 // used only to assign ids to tx
 	// vip_index map[string]map[int64]map[string]map[int64]*b.Tree
+
+	vrw sync.RWMutex
```

Comment:
Please use [mutex hat](https://dmitri.shuralyov.com/idiomatic-go/entries/2), so it's clear what these mutexes protect.

Reply:
I'll fix it, thanks